templates/oauth_result.html.njk

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
```

Comment:
Why are the html, head and body tags omitted?

Reply:
https://google.github.io/styleguide/htmlcssguide.html#Optional_Tags

The general wisdom seems to be that if you are targeting HTML5 supporting browsers, you can and should omit the tags. I don't think we need to support IE since Riot doesn't, which makes this thing a bandwidth saver.

Reply:
ok... i guess.. nothing wrong
I'll try to ignore that these tips come from the company bringing us Chrome and AMP.

Reply:
Hah, definitely healthy to have scepticism where Google is concerned but I think this is a safe suggestion :)